scipy/ndimage/tests/test_filters.py

```diff
@@ -692,91 +693,75 @@ def test_gauss_memory_overlap(self):
         ndimage.gaussian_filter(input, 1.0, output=input)
         assert_array_almost_equal(output1, input)
 
-    @pytest.mark.parametrize(
-        'function_name', ['gaussian_filter', 'uniform_filter',
-                          'minimum_filter', 'maximum_filter'])
+    @pytest.mark.parametrize(('filter_func', 'size0', 'size'),
+                             [(ndimage.gaussian_filter, 0, 1.0),
+                              (ndimage.uniform_filter, 1, 3.0),
+                              (ndimage.minimum_filter, 1, 3.0),
+                              (ndimage.maximum_filter, 1, 3.0)])
```

Comment:
I think these are supposed to be integers. (Apparently, this works with floats that are integral.)
```suggestion
                              (ndimage.uniform_filter, 1, 3),
                              (ndimage.minimum_filter, 1, 3),
                              (ndimage.maximum_filter, 1, 3)])
```